src/frontend/src/flows/pin/idb.ts

```diff
@@ -11,10 +11,8 @@ export {
 /* IndexedDB-specific storage for browser identities
  * (indexed by user number) */
 
-export const idbIdentitiesStore = createStore(
-  "browser-identities",
-  "identities"
-);
+export const idbIdentitiesStore = () =>
```

Comment:
that has an impact at runtime, if there would be a way to avoid it, I would not be against

also did you check it still works fine in Safari? No other reason of asking than my lack of trust in that browser given my experience using idb in safari

Reply:
>  if there would be a way to avoid it, I would not be against

I couldn't find one. The issue is that this file is imported before `test-setup` is run, meaning the `createStore` call is made before `global.indexeddb` is set (using `fake-indexeddb`). We could e.g. create a singleton or something, but I wonder if this is really worth the very tiny performance penalty.

Suggestions welcome!

Reply:
Gotcha.